MH4U Dex/CoreDataController.m

```diff
@@ -46,88 +48,79 @@ + (CoreDataController *)sharedCDController
 
 #pragma mark - Initial Data Loading
 
+- (void)tryLoadSequence
+{
+    if ([[NSUserDefaults standardUserDefaults] boolForKey:NeedsReloadKey]) {
+        NSLog(@"Resetting Core Data");
+        [self resetCoreData];
```

Comment:
Couple thoughts here: 

a) You will likely want to use `DLog` rather than `NSLog` to prevent log barf in the release version. 
b) If you need to log these things, would it make more sense to put those logs in the individual methods so that you can see every time they're getting called rather than just when they're called in this load sequence?